testing/percona-xtradb-cluster-galera/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtradb-cluster-galera
+pkgver=5.7.16
+_pkgver=5.7.16-27.19
```

Comment:
The problem here is that perconas version number works like this, mysql.version-wsrep.version. Sometimes they bump the wsrep version but not the mysql version, and sometimes the other way around. I wanted to add the whole version number to pkgver but abuild does not like dashes. What would be the best solution for this?